examples/scripts/compute.py

```diff
@@ -84,7 +85,8 @@
     # extract parameters and results
     params = parameter_as_dict(es)
     results = m.results()
```

Comment:
can this be deleted now because the results parameter is not used?

Reply:
Yes after the PR is finalized we can do cleanup